app/scripts/views/mixins/experiment-mixin.js

```diff
@@ -33,24 +33,32 @@ define(function (require, exports, module) {
     },
 
     /**
-     * Is the user in an experiment?
+     * Create an experiment and add it to the list of active experiments.
+     * Only creates an experiment with `experimentName` once.
+     * This is useful to create an experiment that is not created
+     * at startup.
      *
-     * @param {String} experimentName
-     * @return {Boolean}
+     * @param {String} experimentName - name of experiment to create.
+     * @param {String} groupType - which group the user is in.
+     * @returns {Object} experiment object, if created.
      */
-    isInExperiment (...args) {
-      return this.experiments.isInExperiment(...args);
-    },
+    createExperiment (...args) {
+      // force the flow model to be initialized so that
+      // the experiment is logged.
+      this.notifier.trigger('flow.initialize');
```

Comment:
I wonder if this is the right place to do this, or whether this should be taken care of in the connect-another-device-mixin or within lib/experiment itself.